irohad/consensus/consensus_common.hpp

```diff
@@ -25,6 +25,7 @@
 #include "model/peer.hpp"
 #include "model/signature.hpp"
 
+// TODO: this file's naming and location are odd.
```

Comment:
Could you provide more details? The naming was supposed to reflect that this header is used in different implementations of consensus algorithms. It is also used both in consensus algorithm, and validation.